src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,14 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
```suggestion
  const { name, email } = user;
```
There is no need to split the line, as you have only two props